app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -177,6 +152,32 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
     override fun onResume() {
         super.onResume()
         TelemetryIntegration.INSTANCE.startSession(this)
+
+        val safeIntent = intent.toSafeIntent()
+        // Skip onboarding if turbo mode is set via intent. This is used in automated perf testing.
```

Comment:
super-nit: I prefer when the comments are on top of a code block (though this may be many years of convention) and I haven't seen many code bases where comments in between code are a valid convention. This isn't well thought through but I think:
- the comments are more noticeable at the top
- If a block of code is so confusing that it needs a comment on its own line, that block of code should be grouped separately from nearby code. However, I often see:
```kotlin
unrelatedCode.doesSomethingUnrelated()
// Comment for a confusing block of code
confusingCode.doesSomethingConfusing()
moreLinesOfConfusingCode.goHere()
```

In our particular case, the comment is applying to the entire block of code, rather than to the if statement it's situated above so to me it makes more sense to put it at the top of the block.

Reply:
Spoke about this in person.  I find the comment to apply to the following line, but I can see how it does also somewhat apply to the entire block.  I'm happy to move it.